pages/terms/term-en-md.js

```diff
@@ -1,166 +1,229 @@
-<ol>
-<li>limit or exclude our or your liability for death or personal injury resulting from negligence;</li>
-<li>limit or exclude our or your liability for fraud or fraudulent misrepresentation;</li>
-<li>limit any of our or your liabilities in any way that is not permitted under applicable law; or</li>
-<li>exclude any of our or your liabilities that may not be excluded under applicable law.</li>
-</ol>
-<p>The limitations and exclusions of liability set out in this Section and elsewhere in this disclaimer: (a)
-are subject to the preceding paragraph; and (b) govern all liabilities arising under the disclaimer or
-in relation to the subject matter of this disclaimer, including liabilities arising in contract, in tort
-(including negligence) and for breach of statutory duty.</p>
-<p>To the extent that the website and the information and services on the website are provided free of charge,
-we will not be liable for any loss or damage of any nature.</p>
-<h2>You must attend your interested Opportunity </h2>
-<p>You will have to attend the Opportunity that you have said yes too. Give 42 hours notice to Teacher or Activity provider if you cant make it.</p>
-<h2>Be honest </h2>
-<p>Dont lie its not cool mate.</p>
-<h2>Buy Ete KFC</h2>
-<p>Buy Ete KFC yasssss</p>
-<h2>Give Ete . KMart Voucher!</h2>
-<p>KMART for life!!!!!!!</p>
-<h2>Credit & Contact Information</h2>
-<p>This Terms and conditions page was created at <a style="color:inherit;text-decoration:none;cursor:text;"
-href="https://termsandconditionstemplate.com">termsandconditionstemplate.com</a> generator. If you have
-any queries regarding any of our terms, please contact us.</p>
-<br/><br/>
-p>The limitations and exclusions of liability set out in this Section and elsewhere in this disclaimer: (a)
-are subject to the preceding paragraph; and (b) govern all liabilities arising under the disclaimer or
-in relation to the subject matter of this disclaimer, including liabilities arising in contract, in tort
-(including negligence) and for breach of statutory duty.</p>
-<p>To the extent that the website and the information and services on the website are provided free of charge,
-we will not be liable for any loss or damage of any nature.</p>
-<h2>You must attend your interested Opportunity </h2>
-<p>You will have to attend the Opportunity that you have said yes too. Give 42 hours notice to Teacher or Activity provider if you cant make it.</p>
-<h2>Be honest </h2>
-<p>Dont lie its not cool mate.</p>
-<h2>Buy Ete KFC</h2>
-<p>Buy Ete KFC yasssss</p>
-<h2>Give Ete . KMart Voucher!</h2>
-<p>KMART for life!!!!!!!</p>
-<h2>Credit & Contact Information</h2>
-<p>This Terms and conditions page was created at <a style="color:inherit;text-decoration:none;cursor:text;"
-href="https://termsandconditionstemplate.com">termsandconditionstemplate.com</a> generator. If you have
-any queries regarding any of our terms, please contact us.</p> `
+
```

Comment:
A lot of empty whitespace lines have been introduced here

Reply:
Hmmm all my comments are getting deleted on this PR thread